python/pylibcudf/pylibcudf/io/avro.pyi

```diff
@@ -1,11 +1,16 @@
 # Copyright (c) 2024, NVIDIA CORPORATION.
 from pylibcudf.io.types import SourceInfo, TableWithMetadata
 
-__all__ = ["read_avro"]
-
-def read_avro(
-    source_info: SourceInfo,
-    columns: list[str] | None = None,
-    skip_rows: int = 0,
-    num_rows: int = -1,
-) -> TableWithMetadata: ...
+__all__ = ["AvroReaderOptions", "AvroReaderOptionsBuilder", "read_avro"]
```

Comment:
I don't see `__all__`'s in other type stub files. Should it be included?

Reply:
IMO yes, ideally the stubs should also have `__all__`

Reply:
Ok, will follow up to add `__all__`